org.opent2t.sample.hub.superpopular/com.smartthings.hub/js/thingTranslator.js

```diff
@@ -161,6 +165,24 @@ class Translator {
         });
     }
 
+    _subscribe(deviceId) {
```

Comment:
[https://github.com/openT2T/translators/issues/137](https://github.com/openT2T/translators/issues/137)


---
In reply to: [90439673](https://github.com/openT2T/translators/pull/134#discussion_r90439673) [](ancestors = 90439673,90384719)